src/main/java/org/jahia/commons/encryption/EncryptionUtils.java

```diff
@@ -147,4 +151,70 @@ public static String sha1DigestLegacy(String source) {
     private EncryptionUtils() {
         super();
     }
+
+    /**
+     * Allows applications to initialize the encryptor configuration before first use.
+     * This method should be called during application startup, before any encryption operations.
+     *
+     * @param password the encryption password (optional, will use config/default if null)
+     * @param algorithm the encryption algorithm (optional, will use config/default if null)
+     * @throws IllegalStateException if the encryptor is already initialized
+     */
+    public static void initializeEncryptor(String password, String algorithm) {
+        initializeEncryptor(password, algorithm, false);
+    }
+
+    /**
+     * Allows applications to initialize the encryptor configuration before first use.
+     * This method should be called during application startup, before any encryption operations.
+     *
+     * <p><strong>WARNING:</strong> Using the force parameter to reinitialize an encryptor that has
```

Comment:
Should we mention somewhere that setting the env variables / Jahia properties after any encryption operation is performed will make the old data undecryptable? Or maybe that will be done on an Academy page?